lib/formats/gzip.js

```diff
@@ -2,6 +2,8 @@ const fs = require('node:fs');
 const fsp = require('node:fs/promises');
 const zlib = require('node:zlib');
 const tar = require('tar-stream');
+const stream = require('node:stream/promises');
+const Promise = require('bluebird');
```

Comment:
Built-in concurrency, already an apostrophe dependency so cheap to add here as well